CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ cmake_minimum_required(VERSION 3.8)
 set( LRS_TARGET realsense2 )
 project( ${LRS_TARGET} LANGUAGES CXX C )
 
+option(BUILD_SHARED_LIBS "Build using shared libraries" OFF)
```

Comment:
We already have such definition, why do we need another?
https://github.com/IntelRealSense/librealsense/blob/e1688cc318457f7dd57abcdbedd3398062db3009/CMake/lrs_options.cmake#L9